src/network/PeerTracker_TEST.cc

```diff
@@ -281,11 +282,11 @@ TEST(PeerTracker, PartitionedEnv)
   ignition::common::Console::SetVerbosity(4);
   EventManager eventMgr;
 
-  setenv("IGN_PARTITION", "p1", 1);
+  ignition::common::setenv("IGN_PARTITION", "p1");
   auto tracker1 = PeerTracker(
       PeerInfo(NetworkRole::SimulationPrimary), &eventMgr);
 
-  setenv("IGN_PARTITION", "p2", 1);
+  ignition::common:setenv("IGN_PARTITION", "p2");
```

Comment:
There's something funky here that's messing up CI:

```
/var/lib/jenkins/workspace/ignition_gazebo-ci-pr_any-ubuntu_auto-amd64/ign-gazebo/src/network/PeerTracker_TEST.cc:289:19: error: expected primary-expression before ���:��� token
   ignition::common:setenv("IGN_PARTITION", "p2");
```

Reply:
4dbb0ac